### ISSUE TITLE
Normalize error sections in DOM

### ISSUE THREAD

**Issue by nielsdos** [php/doc-en#4225]
Many DOM error section just list the constant for the $code for DOMException, but don't actually say anywhere that it's a DOMException being thrown.

**Comment by ypoleto** [comment]
Hey, Is there an example of error section with the problem?

**Comment by nielsdos** [comment]
Hi

If we take a look at this manual page https://www.php.net/manual/en/domcharacterdata.substringdata.php under the "Errors/Exceptions" section it says:

> [DOM_INDEX_SIZE_ERR](https://www.php.net/manual/en/dom.constants.php#constant.dom-index-size-err)
>
>    Raised if offset is negative or greater than the number of 16-bit units in data, or if count is negative.

However, it's not clear that this is a code within the DOMException class.
Annoyingly, the old DOM classes also allow to use warnings instead of exceptions, so this does not _always_ throw and it depends on how the document was configured.
I'm not too sure myself how to clearly document this.

**Comment by ypoleto** [comment]
Do you have any ideia of how to start this issue? I'm not familiar with DOM

**Comment by ypoleto** [comment]
I want to start this issue but I'm not very sure about how to start, do you have any ideas?

**Comment by Girgias** [comment]
I don't think this is the best issue to start working on the documentation. I would recommend looking at one of the other [Good First Issues](https://github.com/php/doc-en/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to get familiar with the documentation conventions and the build process.
